src/dealDashboard/discussionThread/discussionThread.scss

```diff
@@ -0,0 +1,135 @@
+@import "styles/styles.scss";
+@import "src/resources/elements/primeDesignSystem/styles";
+
+.thread {
```

Comment:
to minimize chances of leaking classnames, I might call this something more specific, like ".discussionThread".